### ISSUE TITLE
Cerebellum not segmented in T1/T2* volumes

### ISSUE THREAD

**Issue by dylanablack** [VitoLin21/Rodent-Skull-Stripping#3]
Hi, this is a great tool, thank you! I am attempting skull-stripping in mouse T1/T2*-weighted FLASH volumes and the mask does not include much of the cerebellum. This might be due to differences in contrast between my data and the training data. Is there a plan for the project to fine-tune the pre-trained model on user data?

**Comment by VitoLin21** [comment]
Thank you! We are very glad to receive your suggestion. Currently, due to time constraints, we have no intention of adding the fine-tuning function for the time being. We will consider adding this function when we have time next year. If you need it now, you can use our publicly available data combined with your own data for retraining.
